modules/navigation/nav_agent.cpp

```diff
@@ -134,9 +139,12 @@ void NavAgent::dispatch_avoidance_callback() {
 
 	Vector3 new_velocity;
 
+#ifndef _3D_DISABLED
 	if (use_3d_avoidance) {
 		new_velocity = Vector3(rvo_agent_3d.velocity_.x(), rvo_agent_3d.velocity_.y(), rvo_agent_3d.velocity_.z());
-	} else {
+	} else
+#endif
+	{
 		new_velocity = Vector3(rvo_agent_2d.velocity_.x(), 0.0, rvo_agent_2d.velocity_.y());
 	}
```

Comment:
When enabled it's:
```
	} else
	{
		...
	}
```

Reply:
Ok, I figured I was just reading it wrong.  I thought we were excluding the if above and only would not have the 'else since the #endif was below the 'else'.  Thanks!

Reply:
When we do enable 3D it includes the `} else`, otherwise it just starts with a `{`